package.json

```diff
@@ -48,7 +48,6 @@
 		"vscode-languageserver-textdocument": "^1.0.7"
 	},
 	"devDependencies": {
-		"@types/jest": "^28.1.3",
```

Comment:
Is there a particular reason this needed removing? Thought it'd be necessary so we get type help from the VSCode TS lang server

Reply:
`@types/jest` only adds jest functions to global namespace and it has clashing type definitions with `@jest/globals`, so it's easier to just remove `@types/jest` and import the needed functions from `@jest/globals`.

The issue talking about the issue seems to be resolved but I didn't have any luck updating the dependencies.
https://github.com/facebook/jest/issues/13199

Reply:
That's a good reason - thanks for the explanation :)